### DISCUSSION TITLE
02. What is the 'why' for using open source solutions in pharma clinical data analytics?

### DISCUSSION THREAD

**Discussion by MichaelRimler** [phuse-org/OSTCDA#2] [General]
What is the 'why' for using open source solutions in pharma clinical data analytics?

**Comment by MichaelRimler** [comment]
- More agility to solve problems
- Solutions to new problems available quickly
- Features determined by user needs/requests, not by ROI
- Community development - solutions that may better fit the user base
- Transparency of underlying code - allows risk assessment by users (accuracy)
- Academia is using open source languages more often in their curricula (e.g., R, python) vs commercially available languages - future workforce
-- Biostats, Data science, Epidemiology/public health


PHUSE Blog
https://phuse.global/Communications/PHUSE_Blog/why-r

**Comment by rossfarrugia** [comment]
- Ability to provide community maintenance/support/testing
- Opens us up to making use of other advancements, e.g. LLM - see https://phuse.s3.eu-central-1.amazonaws.com/Archive/2023/SDE/APAC/Bengaluru/PRE_Bengaluru03.pdf 
- Industry collaboration could lead to increased harmonization of the way we work
- Potential to smooth submissions, given greater trust/acceptance could be accredited to commonly used open source code over time - and the greater transparency (as mentioned above) should mean less health authority questions like "can you send me the exact source code and method used to create this p-value?"
